spec/unit/resources/splunk_conf_spec.rb

```diff
@@ -4,8 +4,8 @@
 include CernerSplunk::ConfHelpers, CernerSplunk::ResourceHelpers
 
 shared_examples 'splunk_conf' do |platform, version, package|
-  describe 'action :configure' do
-    let(:runner_params) { { platform: platform, version: version, user: 'root' } }
+  let(:runner_params) { { platform: platform, version: version, user: 'root' } }
+  chef_context 'action :configure' do
```

Comment:
952a76fd23eb7d3cc8b19ec62d1d463c2bda6ab6

I would've aliased the method to make it simpler, but I'm using context in the method implementation and contexts can't be used top level.